src/main/java/uk/gov/ida/verifyserviceprovider/services/EidasAssertionService.java

```diff
@@ -88,7 +92,7 @@ public LevelOfAssurance extractLevelOfAssuranceFrom(Assertion countryAssertion)
     }
 
     public Boolean isCountryAssertion(Assertion assertion) {
-        return metadataResolverRepository.getResolverEntityIds().contains(assertion.getIssuer().getValue());
+        return isEnabled && metadataResolverRepository.get().getResolverEntityIds().contains(assertion.getIssuer().getValue());
```

Comment:
I don't think this should block the PR, but I think we could use something similar to the [nullobject pattern](https://en.wikipedia.org/wiki/Null_object_pattern) here so that we don't need to pass Optional arguments and booleans everywhere.